Cargo.toml

```diff
@@ -156,3 +156,12 @@ hotshot-testing ={ git = "https://www.github.com/EspressoSystems/HotShot.git", t
 hotshot-types = { git = "https://www.github.com/EspressoSystems/HotShot.git", tag = "0.5.75-patch1" }
 libp2p-networking = { git = "https://github.com/EspressoSystems//HotShot.git", tag = "0.5.75-patch1" }
 hotshot-example-types = { git = "https://www.github.com/EspressoSystems/HotShot.git", tag = "0.5.75-patch1" }
+
+[profile.dev]
+opt-level = 1
+# Skip compiling the debug information in the first place.
+debug = 0
+# Skip linking debug information.
+strip = "debuginfo"
```

Comment:
is there a reason not to just do `strip = true`? (I don't actually know how significant this is)

Reply:
I know that we don't want `debuginfo`. I'm not sure if we want `symbols` or not.